data/json/npcs/exodii/exodii_merchant_talk.json

```diff
@@ -638,6 +648,65 @@
+    "responses": [
+      { "text": "Why do they act like zombies when they \"malfunction\"?", "topic": "TALK_EXODII_MERCHANT_Robot_Make" },
+      { "text": "Alright.  Can we talk about something else?", "topic": "TALK_EXODII_MERCHANT_Talk" },
+      { "text": "Well, I'd better be going.  Bye.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "id": "TALK_EXODII_MERCHANT_Robot_Make",
+    "type": "talk_topic",
+    "dynamic_line": ".",
+    "//~": "It has to do with how we make them.  Our technology isn't exactly like yours in the sense of robots, as much of it is cobbled together.  Despite all the stuff we can do, it's very difficult to make the central processing units, and they're pretty basic compared to other robotics.  They're too basic for advanced work, but they take orders very well.  We slap them onto whatever chassis we need at the time, start them up, and set them to work.  Usually, they act very strange when that processor is damaged.",
+    "responses": [
+      { "text": "How long does it take you to make one of these robots?", "topic": "TALK_EXODII_MERCHANT_Robot_Make_Time" },
+      { "text": "Can I buy one of your machines?", "topic": "TALK_EXODII_MERCHANT_Robot_Buy" },
+      { "text": "Can we talk about something else?", "topic": "TALK_EXODII_MERCHANT_Talk" },
+      { "text": "Well, I'd better be going.  Bye.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "id": "TALK_EXODII_MERCHANT_Robot_Make_People",
+    "type": "talk_topic",
+    "dynamic_line": ".",
+    "//~": "The player shouldn't see this unless they fully join the Exodii, as that's when they could be on the inside of things.  For now, this should only link to special dialouge when you've built up enough trust with them.",
+    "//2": "They're not people anymore, the brain's too damaged.  Trust me, we tried that a few times, it didn't turn out so well.  They're what you call, \"ferals\"?  You know, those zombies that act like they're still people, but aren't.",
+    "responses": [
+      { "text": "How long does it take you to make one of these robots?", "topic": "TALK_EXODII_MERCHANT_Robot_Make_Time" },
+      { "text": "Can I buy one of your machines?", "topic": "TALK_EXODII_MERCHANT_Robot_Buy" },
+      { "text": "Can we talk about something else?", "topic": "TALK_EXODII_MERCHANT_Talk" },
+      { "text": "Well, I'd better be going.  Bye.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "id": "TALK_EXODII_MERCHANT_Robot_Make_Time",
+    "type": "talk_topic",
+    "dynamic_line": ".",
+    "//~": "It can take a while, the hardest part is sourcing the materials for one.  We've got printers that make the body, and others that make the tools and equipment the robot will use.  Some of the stuff we make, like the core processor, has to be manufactured by hand, so we have a small team that does that.",
```

Comment:
```suggestion
    "//~": "Faster than you might guess, but it can still take a while, the hardest part is sourcing the materials for one.  We've got printers that make the body, and others that make the tools and equipment the walker will use.  Some of the special parts have to be manufactured by hand, so we have a small team that does that.  Right now, it's very slow, we're still setting up our operations.",
```